taxcalc/consumption.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_MPC_e17500": {
+        "start_year": 2013,
+        "long_name": "Marginal propensity to consume medical expenses",
+        "description": "Defined as dollar change in medical-expense consumption divided by dollar change in income.  Must be in [0,1] range.",
```

Comment:
This seems to me like the kind of thing that would be better to control with code/comments close to the input mechanisms, rather than within the calculation itself. If you run `_validate_mpc_values` within the `Consumption` class, a user can never use the code to model the cases Matt suggested. If instead you just mention that 1 = 100% in `consumption.json` and leave the input control to the various consumers of taxcalc (e.g. `taxbrain/consumption.py` and `webapp-public`), then they can decide between guarding against user error in this way or offering increased functionality. `webapp-public` for example has a lot more options for ensuring users mean what they send with its web GUI.